### PR TITLE
fix issues found during release process

### PR DESCRIPTION
**Pull request by Azulinho**

### CODE REVIEW

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 1]
```diff
@@ -85,6 +83,13 @@ Preparing For a Release
 
    This uses SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
 
+   Add your SSH key to the ``sshd`` agent.
+
+   .. prompt:: bash $
+
+      [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
+      ssh-add $HOME/.ssh/id_rsa
```
Comment (adamtheturtle) [inline docs/gettinginvolved/infrastructure/release-process.rst]: Do this only if the file exists, and mention explicitly that it should be changed to a key which is linked to your GitHub account.

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 2]
```diff
@@ -118,7 +123,8 @@ Preparing For a Release
 
    - The NEWS date format is YYYY-MM-DD.
    - The NEWS file should also be updated for each pre-release and Weekly Development Release, however there should be only one NEWS entry for each Major Marketing Release and Minor Marketing Release.
-   - This means that in doing a release, you may have to change the NEWS heading from a previous Weekly Development Release or pre-release.
+     This means that in doing a release, you may have to remove the previous development release or pre-release header, merging the changes from that previous release into the current release.
+     This applies to both Weekly Development Releases or pre-releases.
```
Comment (adamtheturtle) [inline docs/gettinginvolved/infrastructure/release-process.rst]: "This applies to both Weekly Development Releases or pre-releases." is redundant.

**Comment by adamtheturtle** [comment]
Great, thanks. Please address, and merge when the docs builder is green.

File: docs/gettinginvolved/infrastructure/release-process.rst (hunk 1 above)
Comment (Azulinho) [inline docs/gettinginvolved/infrastructure/release-process.rst]: ssh -A won't work if ssh-agent is not running and the key added to the agent. So it's not really optional here.

the github comment is spot on

**Pull request merged**